docs/custom_loading.rst

```diff
@@ -9,6 +9,39 @@ Specutils leverages the astropy io registry to provide an interface for convenie
 loading data from files. To create a custom loader, the user must define it in
 a separate python file and place the file in their ``~/.specutils`` directory.
 
+Loading from a FITS File
+------------------------
+A spectra with a *Linear Wavelength Solution* can be read using the `Spectrum1D.read`
+function to parse the file name and format
+
+
+.. code-block:: python
+
+  import os
+  from specutils import Spectrum1D
+
+  file_path = os.path.join('path/to/folder', 'file_with_1d_wcs.fits')
+
+  spec = Spectrum1D.read(file_path, format='wcs1d-fits')
+
+
+This will create a `Spectrum1D` object that you can manipulate later.
+
+For instance, you could plot the spectrum.
+
+.. code-block:: python
+
+  import matplotlib.pyplot as plt
+
+  plt.title('FITS file with 1D WCS')
```

Comment:
for end-to-end plotting examples, you can use the `.. plot::` directive rather than hard code an image file, example is e.g. here in the astropy docs: https://github.com/astropy/astropy/blob/master/docs/visualization/lupton_rgb.rst

Full docs: https://matplotlib.org/devel/plot_directive.html

Reply:
Awesome! will do

Reply:
@bsipocz I managed to create the plot using the `.. plot::` directive. But I was stopped in how to reference the file that I want to read. I found that `specutils/tests/test_loaders.py` uses `astropy.utils.data.get_pkg_data_filename`. I used it in the same way but did not work. In a previous PR I used something like:

```python 
import os
file_path = os.path.join(os.path.dirname(__file__),
                         'data/L5g_0355+11_Cruz09.fits')
```
This was _objected_ by @crawfordsm in #188  (which, by the way, the correct answer is above) 

Finally I noticed that the other plots in specutils' documentation are hard coded as well.

Reply:
the flag `# doctest: +REMOTE_DATA` need to be added for those doc examples using a remote data file. Howto examples are in the docs: http://docs.astropy.org/en/stable/development/testguide.html#working-with-data-files, or working examples are in e.g. the photutils' docs files (e.g https://github.com/astropy/photutils/blob/master/docs/getting_started.rst). 

But anyway, This PR good as is, and and issue can be opened as a reminder to switch to full, testable examples using the plot directive as a follow-up, including factoring out all the other hard coded image files.